phases/1-programming.md

```diff
@@ -46,6 +69,7 @@ Read these books and add a 100 - 200 word review to your weekly journal. This sh
 
 [Getting Real]: http://gettingreal.37signals.com/
 [Passionate Programmer]: http://www.amazon.com/The-Passionate-Programmer-Remarkable-Development/dp/1934356344
+[Pragmatic Programmer]: https://www.amazon.com/Pragmatic-Programmer-Journeyman-Master/dp/020161622X
```

Comment:
do we own a copy of this one, @cromwellryan ? We might need to get one or two.

Reply:
Seems we don't. Let's get a digital copy. Leah can help with that.